src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlNodeList.cs

```diff
@@ -80,7 +80,7 @@ public bool IsReadOnly
             get { return _nodeArray.IsReadOnly; }
         }
 
-        object IList.this[int index]
+        object? IList.this[int index]
```

Comment:
is it actually possible for this to return null? we're throwing when user tries to add null

Reply:
It doesn't look like it, but it must match the index operator for `IList`

Reply:
I don't expect this to be fixed, just unresolving as FYI:
for return values you can make it non-nullable even if interface returns nullable (for setter it has to match but you can make it non-nullable with `[AllowNull]`)